models/api.models.js

```diff
@@ -15,12 +15,28 @@ exports.fetchTopics = () => {
 	});
 };
 
+exports.fetchArticles = () => {
+	let sqlQuery = `
+      SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, 
+      COUNT(comments.comment_id)::int AS comment_count 
+      FROM articles 
+      LEFT JOIN comments ON comments.article_id = articles.article_id 
+      GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url ORDER BY articles.created_at DESC`;
```

Comment:
It doesn't need to GROUP BY so many columns, only by articles.article_id will do the work

Reply:
Also, although `:: int` will do the work, keeping it uppercase to keep the standard syntax would be better and less likely to cause any future issues `:: INT`